lib/rules/no-curly-component-invocation.js

```diff
@@ -44,6 +44,8 @@ const BUILT_INS = [
   'in-element',
 ];
 
+const ALWAYS_CURLY = ['yield'];
```

Comment:
I guess we can remove it from line 42 now, right?

Reply:
Good call. I can do that.